### PR TITLE
PP-10406: Add missing docs

### PR DESCRIPTION
**Pull request by jkadluczka**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira]()

# Description

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/monetization/partner-program-api/index.mdx [hunk 1]
```diff
@@ -3989,6 +4287,55 @@ curl --request POST \
 
 </Section>
 
+<Section>
+
+<Text>
+
+## Get Solution Summary
+
+`POST /v2/partners/solutions/summary` - get summary from solution program
```
Comment (ale-kacperczyk) [inline src/pages/monetization/partner-program-api/index.mdx:4296]:
```suggestion
`POST /v2/partners/solutions/summary` - get summary from Solution Program
```

**Review by ale-kacperczyk — COMMENTED** [review]
I don't think we need to commit changes in package-lock

**Comment by jkadluczka** [comment]
> I don't think we need to commit changes in package-lock

Yeah, we dont- it should be in gitignore. I will revert the changes.

File: src/pages/monetization/partner-program-api/index.mdx [hunk 2]
```diff
@@ -548,6 +550,109 @@ curl --request GET \
...
+
+</Section>
+
+<Section>
+
+<Text>
+
+## Get Commission Metric
+
+`GET /v2/partners/metric/commission` - get commission metric
```
Comment (fabcos92) [inline src/pages/monetization/partner-program-api/index.mdx:607]:
```suggestion
`GET /v2/partners/metrics/commission` - get commission metric
```

**Review by hubert-wyszynski — APPROVED** [review]

File: src/pages/monetization/partner-program-api/index.mdx [hunk 3]
```diff
@@ -548,6 +550,109 @@ curl --request GET \
...
+<CodeResponse title={'EXAMPLE RESPONSE - 200'}>
+
+```json
+{
+  "pending": 21.00,
+  "withdrawn": 37.00,
+  "available": 362.00,
+  "total": "420.00",
+}
+```
```
Comment (fabcos92) [inline src/pages/monetization/partner-program-api/index.mdx:648]: Commission metric returns different structure:
```
{
    "value": 47022.6
}
```

Comment (jkadluczka) [reply src/pages/monetization/partner-program-api/index.mdx:648]: Damn, my bad 🤦

**Review by fabcos92 — APPROVED** [review]

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**